### ISSUE TITLE
Ignore certain files or directories from being built into artifact

### ISSUE THREAD

**Issue by kevinquillen** [acquia/blt#1499]
My system information:
* Operating system version: macOS
* BLT version: 8.6.15

Is there a way to specify the exclusion of certain files or directories from the build artifact? I added a `deploy-exclude-additions.txt` file under `/blt` but it does not seem to remove the files from deploy.

**Comment by dpagini** [comment]
It looks like the `deploy-exclude-additions.txt` file was not yet available in BLT v8.6.15. I would say your options are to either upgrade BLT (looks like this became available in 8.8, maybe?), or to simply use the `deploy-exclude` custom file.

I have been using the second option, and can give some suggestions on that...
* Copy the `vendor/acquia/blt/phing/files/deploy-exclude.txt` file to somewhere in your project, with the most common place being `blt/phing/files/deploy-exclude.txt` (or something similar).
* Update your `blt/project.yml` to point to the new file. That probably involves adding the following:
```yml
deploy:
  exclude_file: ${repo.root}/blt/phing/files/deploy-exclude.txt
```
* customize your copied file with your custom deployment excludes

Let me know how that works for you! Good luck!

**Comment by grasmash** [comment]
Closing. Thanks @dpagini for your help.
